### ISSUE TITLE
What's up with the colors?

### ISSUE THREAD

**Issue by johnpaulbin** [vipermu/StyleCLIP#1]
Putting anything will give a pretty good output, but in some situations the colors will become all mixed. 

![80](https://user-images.githubusercontent.com/43943237/112529623-b94bdc00-8d62-11eb-887e-c2b3082275cd.png)

Is there any reason behind this?

**Comment by johnpaulbin** [comment]
Also how could I fix if possible ^

**Comment by DuncanRowland** [comment]
In clip_generate.py line 111:
   img = img[0].permute(1, 2, 0).detach().cpu().numpy() * 256
should be
   img = img[0].permute(1, 2, 0).detach().cpu().numpy() * 255
